### DISCUSSION TITLE
[Feature] Add Photos to Existing Shared Links from Library

### DISCUSSION THREAD

**Discussion by Braypolk** [immich-app/immich#9289] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### What
I would like to be able to add photos in my library to existing shared links. 

### Why
This is useful when you share a link and forget to upload some photos, or want to upload photos later. Currently, to link the new photos, you can delete the current share link and make a new one, have 2 share links, or allow uploads and use your filesystem to add the new photos. None of these are great.

### How (visually)
On web, the action "Add to shared link" could be above/below the current action "Add to shared album"

![image](https://github.com/immich-app/immich/assets/22568839/bdc48656-cb42-4ba9-82bb-6d479a360895)

On mobile (I'm using Android, not sure if Apple looks the same), there is currently one "Shared" expanding drop-down that shows your shared albums. This could be changed to 2 expanding dropdowns called "Shared Albums" and the other "Shared Links"
![Photo May 6 2024](https://github.com/immich-app/immich/assets/22568839/a435891b-bbdb-444b-86b8-b22a8c1e11fc)

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by NicholasFlamy** [comment]
How about a feature to convert a shared link to a shared album?

**Reply by Braypolk** [reply]
Ya that's probably a cleaner way to do it, good thinking!
